wpilibc/src/main/native/include/frc/PowerDistribution.h

```diff
@@ -18,6 +23,7 @@ class PowerDistribution : public wpi::Sendable,
                           public wpi::SendableHelper<PowerDistribution> {
  public:
   static constexpr int kDefaultModule = -1;
+  static constexpr double kUpdatePeriod = 0.025;
```

Comment:
```suggestion
  static constexpr units::second_t kUpdatePeriod = 25_ms;
```
Also add `#include <units/time.h>`.